whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/VersionInformation.java

```diff
@@ -2,27 +2,34 @@
 
 import java.util.UUID;
 
+
 public class VersionInformation {
 
```

Comment:
can we rename to NrtmVersionInfo as rest of classes includes Nrtm as prefix and also it will avoid confusion with existing versionInfo